eng/pipelines/templates/build-job.yml

```diff
@@ -4,50 +4,74 @@ parameters:
   archType: ''
   container: ''
   pool: {}
-  isOfficialBuild: ''
+  isOfficialBuild: false
   runTests: true
 
 jobs:
 
-- template: /eng/common/templates/job/job.yml
+- template: /eng/common/templates/jobs/jobs.yml
   parameters:
-    displayName: ${{ format('{0} {1}', parameters.osGroup, parameters.archType) }}
-    name: ${{ format('{0}_{1}', parameters.osGroup, parameters.archType) }}
-    enableTelemetry: ${{ parameters.isOfficialBuild }}
```

Comment:
I must be missing it but where were we passing isOfficialBuild before? Looks like we were just not passing it and always defaulting to empty here right?

Reply:
I'm passing it down here: https://github.com/dotnet/runtimelab/pull/261/files#diff-d71a1a613ea5de9e905a5c6f641d8c4c79800b54c4667cbcbefdfb2f91a898ddR57

and then the default of the parameter is set to false: https://github.com/dotnet/runtimelab/pull/261/files#diff-6a89a2c86af1832dfc215647fe7dd379b4def9cea440cf14ef179b65eb55e94aR7

Reply:
Yeah, I saw that. I meant before this change, we seem to have been using it to set the enableTelemetry field but we weren't actually setting it on the main yml or importing it unless I'm missing something.

Reply:
Ah I see what you're saying. The arcade templates ignore true if it is in an official build.